proto/storage/test.proto

```diff
@@ -134,6 +134,11 @@ message TestSingleUUIDKeyStruct {
   }
 
   bytes bytess = 16;
+
+  oneof oneof_two {
+    string oneof_two_string = 17;
+    int64 oneof_two_int = 18;
+  }
```

Comment:
Maybe it will be good to add it as a separated PR with a test to give 100% guarantee that new code does not break anything